Library/Homebrew/requirements/xorg_requirement.rb

```diff
@@ -0,0 +1,24 @@
+require "requirement"
+
+class XorgRequirement < Requirement
+  fatal true
+  default_formula "linuxbrew/xorg/xorg"
+
+  def initialize(name = "xorg", tags = [])
+    @name = name
+    if /(\d\.)+\d/ === tags.first
+      tags.shift
+    end
+    super(tags)
+  end
+
+  satisfy :build_env => false do
+    Formula["linuxbrew/xorg/xorg"].installed?
+  end
+
+  def message
+    s = "X.Org is required to install this formula."
```

Comment:
I'd suggest a more descriptive message. Maybe something like something like:
`X.Org is required to install this formula. Use: brew tap linuxbrew/xorg && brew install xorg`